test-parachains/adder/collator/src/main.rs

```diff
@@ -137,6 +137,7 @@ fn main() {
 			executable_name: "adder-collator",
 			description: "collator for adder parachain",
 			author: "parity technologies",
+			support_url: "none"
```

Comment:
shouldn't collator issues still be reported to us?

```suggestion
			support_url: "https://github.com/paritytech/polkadot/issues/new",
```